### ISSUE TITLE
missing data

### ISSUE THREAD

**Issue by Pog3k** [deeplearning-mit-tensorflow-keras-tfjs/2020_Zweite_Auflage#1]
Hello, 
It seems that some data are missing.
chap_7/tensorboard/data/fashion  

there should be some *.gz files but the folder is empty. Even on the rheinwerk homepage.
i.e. train-labels-idx1-ubyte.gz  

plz upload (;

**edit**
found them here:
https://github.com/zalandoresearch/fashion-mnist/tree/master/data/fashion

**Comment by alnd01** [comment]
Thanks!
